l2geth/core/vm/ovm_state_manager.go

```diff
@@ -156,6 +158,32 @@ func putContractStorage(evm *EVM, contract *Contract, args map[string]interface{
 	return []interface{}{}, nil
 }
 
+func putAccountCode(evm *EVM, contract *Contract, args map[string]interface{}) ([]interface{}, error) {
+	address, ok := args["_address"].(common.Address)
+	if !ok {
+		return nil, errors.New("Could not parse address arg in putAccountCode")
+	}
+	code, ok := args["_code"].([]byte)
+	if !ok {
+		return nil, errors.New("Could not parse code arg in putAccountCode")
+	}
+
+	// save the block number and address with modified key if it's not an eth_call
+	if evm.Context.EthCallSender == nil {
+		err := evm.StateDB.SetDiffAccount(
+			evm.Context.BlockNumber,
+			address,
+		)
+		if err != nil {
+			log.Error("Cannot set diff key", "err", err)
+		}
+	}
+	evm.StateDB.SetCode(address, code)
+	log.Debug("Put account code", "address", address.Hex(), "code", hex.EncodeToString(code))
```

Comment:
This logline can become very expensive and will pollute the logs with large amounts of code. Could we remove the second half of the logline? `code` + `hex.EncodeToString`